modules/distr-tracing-tempo-config-query-rbac.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr-tracing-tempo-configuring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-configuring-query-rbac_{context}"]
+= Configuring the query RBAC
+
+As an administrator, you can set up the query Role-Based Access Control (RBAC) to filter the span attributes for your users by the namespaces for which you granted them permissions.
+
+[NOTE]
+====
+When you enable the query RBAC, users can still access traces from all namespaces, and the `service.name` and `k8s.namespace.name` attributes are also filtered for all users.
+====
+
+.Prerequisites
+
+* An active {oc-first} session by a cluster administrator with the `cluster-admin` role.
++
+[TIP]
+====
+* Ensure that your {oc-first} version is up to date and matches your {product-title} version.
+
+* Run `oc login`:
++
+[source,terminal]
+----
+$ oc login --username=<your_username>
+----
+====
+
+.Procedure
+
+. Enable multitenancy and query RBAC in the `TempoStack` custom resource (CR), for example:
++
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: simplest
+  namespace: chainsaw-multitenancy
+spec:
+  storage:
+    secret:
+      name: minio
+      type: s3
+  storageSize: 1Gi
+  resources:
+    total:
+      limits:
+        memory: 2Gi
+        cpu: 2000m
+  tenants:
+    mode: openshift
+    authentication:
+      - tenantName: dev
+        tenantId: "1610b0c3-c509-4592-a256-a1871353dbfb"
+  template:
+    gateway:
+      enabled: true # <1>
+    rbac:
+      enabled: true # <2>
```

Comment:
```suggestion
      rbac:
        enabled: true # <2>
```
`rbac` is under `gateway` section: https://github.com/grafana/tempo-operator/blob/d18320b5c1210edd3a093618b07b99087a7d7f51/docs/spec/tempo.grafana.com_tempostacks.yaml#L228

Reply:
Thank you. Fixed.